package.json

```diff
@@ -20,11 +20,12 @@
   "author": "Augusto Lemble <me@augustolemble.com>",
   "license": "ISC",
   "dependencies": {
+    "abi-decoder": "^1.0.8",
     "ethereumjs-abi": "^0.6.4",
     "protobufjs": "^6.0.2",
     "solc": "^0.4.9",
     "truffle-hdwallet-provider": "0.0.3",
-    "zeppelin-solidity": "^1.0.4"
+    "zeppelin-solidity": "1.0.4"
```

Comment:
@nberger if you want to separate it in different PRs you can do that, but if not we can just go ahead and merge this PR. Your choice, but if you want to make it in 3 PRs you will need to do it hahaha.